redirects.json

```diff
@@ -93,167 +103,257 @@
-      "destination": "/design/components/dialog"
+      "destination": "/design/components/dialog/rails/alpha"
     },
     {
       "name": "ViewComponents docs for Primer::Beta::Flash",
       "match": "^view-components/components/beta/flash",
-      "destination": "/design/components/flash"
+      "destination": "/design/components/flash/rails/beta"
     },
     {
       "name": "ViewComponents docs for Primer::Beta::Heading",
       "match": "^view-components/components/beta/heading",
-      "destination": "/design/components/heading"
+      "destination": "/design/components/heading/rails/beta"
+    },
+    {
+      "name": "ViewComponents docs for Primer::Alpha::HiddenTextExpander",
+      "match": "^view-components/components/alpha/hiddentextexpander",
+      "destination": "/design/components/hidden-text-expander/rails/alpha"
     },
     {
       "name": "ViewComponents docs for Primer::Beta::Octicon",
       "match": "^view-components/components/beta/octicon",
-      "destination": "/design/components/icon"
+      "destination": "/design/components/icon/rails/beta"
+    },
+    {
+      "name": "ViewComponents docs for Primer::Beta::IconButton",
+      "match": "^view-components/components/beta/iconbutton",
+      "destination": "/design/components/icon-button/rails/beta"
     },
     {
       "name": "ViewComponents docs for Primer::Alpha::ImageCrop",
       "match": "^view-components/components/alpha/imagecrop",
-      "destination": "/design/components/image-crop"
+      "destination": "/design/components/image-crop/rails/alpha"
     },
     {
       "name": "ViewComponents docs for Primer::Alpha::Image",
       "match": "^view-components/components/alpha/image",
-      "destination": "/design/components/image"
+      "destination": "/design/components/image/rails/alpha"
     },
     {
       "name": "ViewComponents docs for Primer::Beta::Label",
       "match": "^view-components/components/beta/label",
-      "destination": "/design/components/label"
+      "destination": "/design/components/label/rails/beta"
     },
     {
       "name": "ViewComponents docs for Primer::Beta::Link",
       "match": "^view-components/components/beta/link",
-      "destination": "/design/components/link"
+      "destination": "/design/components/link/rails/beta"
     },
     {
       "name": "ViewComponents docs for Primer::Beta::Markdown",
       "match": "^view-components/components/beta/markdown",
-      "destination": "/design/components/markdown"
+      "destination": "/design/components/markdown/rails/beta"
     },
     {
       "name": "ViewComponents docs for Primer::Alpha::NavList",
       "match": "^view-components/components/alpha/navlist",
-      "destination": "/design/components/nav-list"
+      "destination": "/design/components/nav-list/rails/alpha"
+    },
+    {
+      "name": "ViewComponents docs for Primer::Alpha::OcticonSymbols",
+      "match": "^view-components/components/alpha/octiconsymbols",
```

Comment:
This one's destination url is 404'ing for me in staging @camertron

Reply:
@rezrah discovered this was happening due to loose regex match for octicons. It has been fixed in 873f6883045ca554d3f8364673bc41cf37ecd29c

Reply:
👋 @camertron - is it worth pulling this rule out for now and fixing in a follow up PR? The can ship the rest of the redirects now that way.

Reply:
Yeah, at this point I think that's a great idea 👍